### PR TITLE
Create January 2022 service update

### PR DESCRIPTION
**Pull request by B-Bonner**
January 2022 service update.

### Trello card
https://trello.com/c/v1jR6hBt

### Context
January 2022 service update.

### Changes proposed in this pull request
New service update.

### Guidance to review
Check whether the opening line is being pulled into the Service update banner that should display at the top of the dashboard.

You can also view the content by clicking on the Service updates section at the bottom of the dashboard.

### CODE REVIEW

File: data/service_updates/20220124.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+title: 25 January 2022 - Choosing when to close dates to candidates and other features
+summary: We've created a new feature that allows you to choose when to close a school experience date.
+html_content: |-
+  <p>
+    When you add a date, you'll now be able to choose when to close that date to candidates. For example, you might want to close a date 2 days before the event, to give you time to prepare for it.
+  </p>
+
+  <h3>More reaons to choose from when declining a request</h3>
```
Comment (leoapost) [inline data/service_updates/20220124.yml:9]: There's a typo in `More reasons`

Comment (B-Bonner) [reply data/service_updates/20220124.yml]: Thanks!

**Review by leoapost — APPROVED** [review]
:100:

**Comment by jkempster34** [comment]
@B-Bonner Just shipped the dates update 🚢 

I think this service update will need updating to include automatically opening dates, as well as closing them.

**Comment by jkempster34** [comment]
image

image

**Comment by B-Bonner** [comment]
Looks good @jkempster34 Thanks!

**Pull request merged**